app/assets/javascripts/discourse/app/lib/sidebar/base-custom-sidebar-panel.js

```diff
@@ -54,6 +54,24 @@ export default class BaseCustomSidebarPanel {
     return false;
   }
 
+  /**
+   * @returns {boolean} Controls whether the filter is shown
+   */
+  get filterable() {
+    return false;
+  }
```

Comment:
What is the difference between `filterable` and `searchable`? To me they sound like they're the same thing, so maybe we should include a comment here to clarify what each one does?